### PR TITLE
Confirm untitled.is-a.dev for GitHub and Discord

### PR DESCRIPTION
**Pull request by VihaanAnand**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [x] The file is in the `domains` folder and is in the JSON format.
- [x] You have completed your website. 
- [x] The website is reachable.  
- [x] You're not using Vercel or Netlify.  
- [x] The CNAME record doesn't contain `https://` or `/`.  
- [x] There is sufficient information at the `owner` field.  

## Website Link/Preview
[https://untitled.is-a.dev/](https://untitled.is-a.dev/)

### CODE REVIEW

File: domains/_discord.untitled.is-a.dev.json
Comment (Stef-00012) [inline domains/_discord.untitled.is-a.dev.json:1]: file name can't contain .is-a.dev

File: domains/_github-pages-challenge-VihaanAnand.untitled.json
Comment (Stef-00012) [inline domains/_github-pages-challenge-VihaanAnand.untitled.json:1]: file name should be all lowercase, including your github username

**Review by Stef-DP — CHANGES_REQUESTED** [review]

File: domains/_discord.untitled.is-a.dev.json
Comment (VihaanAnand) [reply domains/_discord.untitled.is-a.dev.json:1]: Completed.

File: domains/_github-pages-challenge-VihaanAnand.untitled.json
Comment (VihaanAnand) [reply domains/_github-pages-challenge-VihaanAnand.untitled.json:1]: Completed.

**Review by iostpa — CHANGES_REQUESTED** [review]

**Pull request closed without merging**